numpy/core/src/multiarray/descriptor.c

```diff
@@ -1698,10 +1707,22 @@ PyArray_DescrConverter(PyObject *obj, PyArray_Descr **at)
 NPY_NO_EXPORT PyArray_Descr *
 PyArray_DescrNew(PyArray_Descr *base)
 {
-    PyArray_Descr *newdescr = PyObject_New(PyArray_Descr, &PyArrayDescr_Type);
+    PyObject *tuple = PyTuple_Pack(3,
+                                  /* dtype */ base,
+                                  /* align */ Py_False,
+                                  /* copy  */ Py_True);
+    PyObject *ret = PyObject_CallObject((PyObject*)Py_TYPE(base), tuple);
```

Comment:
`PyObject_CallFunctionObjArgs((PyObject *)Py_TYPE(base), base, Py_False, Py_True)` would be shorter here - or am I missing something?

Reply:
adopted